src/TimeSeries/encodings/timeseriespreprocessing.jl

```diff
@@ -0,0 +1,77 @@
+struct TSStats{T}
+    means::AbstractArray{T,2}
+    stds::AbstractArray{T,2}
```

Comment:
You could also consider changing these to `AbstractMatrix{T}`, but it would be best to confirm first that they will always be 2-dimensional (what about higher-dimensional time series, for example?)

Reply:
Not sure about the higher dimensional series, I will have to check the literature or some examples online if I can find them.

Reply:
Oh I can come up with examples easily enough, the question is whether the fastai docs have any :)

Reply:
Oh, I will check that up before our meeting today.

Reply:
Oh, I will check that up before our meeting today.